frontend/src/metabase/modes/components/drill/AutomaticDashboardDrill.jsx

```diff
@@ -15,10 +16,17 @@ export default ({ question, clicked }: ClickActionProps): ClickAction[] => {
 
   // questions with a breakout
   const dimensions = (clicked && clicked.dimensions) || [];
+
+  // ExpressionDimensions don't work right now (see metabase#16680)
```

Comment:
This function, which is repeated below, looks like a great candidate to live in a shared `util`-type file.

Reply:
this code will get deleted as part of #16680 so I think duplicated logic is fine here